packages/backend/src/peripherals/database/ForcedTransactionsRepository.ts

```diff
@@ -282,11 +282,11 @@ export class ForcedTransactionsRepository extends BaseRepository {
     return !!updated
   }
 
-  async countAll(): Promise<bigint> {
+  async countAll(): Promise<number> {
```

Comment:
The `number` type is precise up to `2^53`. I don't expect to ever have as many records in the database and this greatly simplifies the code in ServerPagination, because you can just use `Math.ceil`